dev_guide/getting_traffic_into_cluster.adoc

```diff
@@ -94,6 +94,21 @@ When a set of routes is created in various projects, the overall set of routes
 is available to the set of routers. Each router admits (or selects) routes from
 the set of routes. By default, all routers admit all routes.
 
+As of {product-title} 3.6, routers can accept IPV6 traffic from outside the cluster for the following routes:
+
+* Route type 1
+* Route type 2
+
+While the traffic is accepted as IPV6, it is internally converted to IPV4 for
+internal use.
```

Comment:
While the traffic is accepted by the router as IPv6, all communications inside the cluster are still done using IPv4.